### ISSUE TITLE
Jetappointments - use service name as the product name on the Woocommerce order info

### ISSUE THREAD

**Issue by Ricarditu** [CrocoBlock/suggestions#7209]
It would be a nice and needed feature to implement for Jetappointment (maybe also JetBooking, not sure because I haven't used it yet)
in which the selected service(s) name would appear as the product(s) name on a Woocommerce order, instead of the base product name.

I give an example:
I have a client which uses JetAppointments with Woocommerce integration and an independent billing software (that communicates with  Woocommerce via their own plugin). The billing software should receive the service name that their cliente booked, instead, they receive the base product name, because that is what's on the Woocommerce order, the base product name, for example "Appointment).

I know the info regarding the service is on the order details and the customer receives that on the email as well, also as details. However, having that information as the product(s) itself makes total sense. That way, the customer knows exactly what service(s) that email/invoice is about instead of a generic name.

Thanks!

**Comment by Ravisi** [comment]
Yes PLEASE! It would be great and is very necessary, I can't generate invoices with the product name "Appointment" and I must to change manually each product name before I generate the invoices, this is really annoying for us. I use Jet Appointments. Please let us use the service name, and it would be even better the time slot as a product meta data.

Thanks!
